live-examples/css-examples/border-width.html

```diff
@@ -0,0 +1,37 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="width">
```

Comment:
I think this should be `borderWidth` (although it doesn't matter much in practice, since all browsers will support both properties).

Reply:
Fixed.